.npmrc

```diff
@@ -1 +1,2 @@
 save-exact=true
+engine-strict=true
```

Comment:
Can we doc in the README why these settings are included and what they do?

Reply:
@gbockus-sf in the main [README](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/README.md)? Or somewhere in developing/contributing?

Reply:
Probably in the contributing docs would be best. good call.  Maybe in developing.md?